src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -72,6 +72,42 @@ const addValidUser = async () => {
   await waitFor(() => expect(nextButton).toBeEnabled());
 };
 
+const addInvalidUser = async () => {
+  const user = userEvent.setup();
+  const addUser = await screen.findByRole('button', { name: /add a user/i });
+  expect(addUser).toBeEnabled();
+  await waitFor(() => user.click(addUser));
+  const enterUserName = screen.getByRole('textbox', {
+    name: /blueprint user name/i,
+  });
+  const nextButton = await getNextButton();
+  await waitFor(() => user.type(enterUserName, '..'));
+  await waitFor(() => expect(enterUserName).toHaveValue('..'));
+  const enterSshKey = await screen.findByRole('textbox', {
+    name: /public SSH key/i,
+  });
+  await waitFor(() => user.type(enterSshKey, 'ssh-rsa d'));
+  await waitFor(() => expect(nextButton).toBeDisabled());
+};
+
+const addInvalidSSHKeyUser = async () => {
```

Comment:
We're already in the User test suite.

```suggestion
const addInvalidSSHKey = async () => {
```